docs/Usage/Cells.md

```diff
@@ -0,0 +1,100 @@
+ # Cells
+ 
+Cells are of three types − Code, Markdown and Raw.
+
+* #### Code Cells
+Contents in this cell are treated as statements in a programming language of current [kernel](../Installation.md#kernels). When a cell is run (See [Hydrogen: Run Cell](GettingStarted.md#hydrogen-run-cell)), its result is displayed inline. 
+* #### Markdown Cells
+These cells contain text formatted using [common markdown](https://www.markdownguide.org/cheat-sheet/). All kinds of formatting features are available like making text bold and italic, displaying ordered or unordered list, rendering tabular contents etc. Markdown cells are especially useful to provide documentation to the computational process.
+* #### Raw Cells
+Contents in raw cells are not evaluated by the kernel. When passed, they will be rendered as desired.
+
+
+## How to create Cells in Hydrogen
+Hydrogen supports a multitude of ways to make cells. Cells are  defined using "Cell markers".
+
+### Cell Markers
+
+A cell marker is a special inline comment that defines the lines that begin each cell. The inline comment must start with the comment character registered in the language's grammar file.
+
+- Python: `#`
+- R: `#`
+- Julia: `#`
+- Javascript: `//`
+- C++: `//`
+- Stata: `//`
+- Matlab: `%`
+
+
+### Example Definitions
+
+#### Code Cells
+
+Code cells are defined, by adding the following markers, after the comment character specefic to the language you are using.
+
+##### Markers:
+- `%% codecell`
+- `<codecell>`
+- `In[]` (you can have any number of charaters inside the brackets (empty also works))
+
+For example, the following text corresponds to three code cells in a Python file
+```py
+# %% codecell
+print('Hello, world!')
+# <codecell>
+print('foo')
+# In[1]
+print('bar')
+```
+The equivalent code in JavaScript will be
+```js
+// %% codecell
+print('Hello, world!');
+// <codecell>
+print('foo');
+// In[1]
+print('bar');
+```
+
+These are the same cell markers used by [**"Hydrogen: Run Cell"**](GettingStarted.md#hydrogen-run-cell).
+
+#### Markdown Cells
+
+You can also create cells with [Markdown](https://www.markdownguide.org/cheat-sheet/)-formatted text. Just as for code cells, there are a range of allowed markers to delimit the start of a markdown cell:
```

Comment:
Remove this first sentence as it was explained above. Maybe link up to the relevant definitions for both cell types though